### PR TITLE
tools: update pd documents

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

File: tools/pd-control.md [hunk 1]
```diff
@@ -338,9 +341,24 @@ Usage:
 ```bash
 >> member                               // Display the information of all members
 {
-  "members": [......],
-  "leader": {......},
-  "etcd_leader": {......},
+  "header": {
+  "leader": {......},	    "cluster_id": 6493707687106161130
```
Comment (rleungx) [inline tools/pd-control.md]: need format here

File: tools/pd-control.md [hunk 2]
```diff
@@ -467,10 +519,19 @@ Use this command to list all Regions of a specific store.
...
+  "regions": [
+    {
+      "id": 8,
+      "start_key": "t\\x80\\x00\\x00\\x00\\x00\\x00\\x00\\xff\\a\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\xf8",
+      ...
+    },
+    {
+      ...
+    }
+  ],
```
Comment (rleungx) [inline tools/pd-control.md]: not comma here

File: tools/pd-control.md [hunk 3]
```diff
@@ -484,7 +545,7 @@ Usage:
 >> region topread
 {
   "count": 16,
-  "regions": [......],
+  "regions": [...],
```
Comment (rleungx) [inline tools/pd-control.md]: no comma in the end

**Comment by disksing** [comment]
PTAL @rleungx

**Pull request merged**